src/python/pants/engine/internals/native.py

```diff
@@ -130,8 +131,18 @@ def lib(self):
     def decompress_tarball(self, tarfile_path, dest_dir):
         return self.lib.decompress_tarball(tarfile_path, dest_dir)
 
-    def init_rust_logging(self, level, log_show_rust_3rdparty: bool, use_color: bool):
-        return self.lib.init_logging(level, log_show_rust_3rdparty, use_color)
+    def init_rust_logging(
+        self,
+        level: int,
+        log_show_rust_3rdparty: bool,
+        use_color: bool,
+        show_target: bool,
+        log_levels_by_target: Dict[str, LogLevel],
```

Comment:
Nit: the type hint for `log_levels_by_target` everywhere can be `Mapping`, given that you convert it into a dict on the next line.

Not a big deal, but can be useful given how frequently we use `FrozenDict`.